next/src/components/ui/SocialLink/SocialLink.module.scss

```diff
@@ -0,0 +1,120 @@
+.link {
+  position: relative;
+
+  .content {
+    position: relative;
+    height: 36px;
+    width: 36px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    z-index: 25;
+    background-color: var(--neutral-200, #efefef);
+    overflow: hidden;
+
+    svg {
+      z-index: 5;
+      width: clamp(16px, calc(20vw / 7.68), 20px);
+      height: clamp(16px, calc(20vw / 7.68), 20px);
+    }
+
+    &::before {
+      content: '';
+      position: absolute;
+      width: 100%;
+      height: 100%;
+      left: -100%;
+      bottom: 0;
+      z-index: 3;
+      transition:
+        transform 0.3s ease-in 0.2s,
+        background-color 0.3s ease-in;
```

Comment:
Consider using some custom `cubic-bezier` easing, that's sharp and fits the style of the design.

Reply:
I'll for sure do that, Don't worry haven't forgotten about it.